### PR TITLE
Sunsetting Shields custom logos

### PR DESCRIPTION
**Pull request by PyvesB**
Goes ahead with the decision made in #9476.

In terms of communication, we could pin the content of the blogs in an issue as well.

### CODE REVIEW

File: doc/logos.md [hunk 1]
```diff
@@ -1,42 +0,0 @@
-# Logos
-
-For documentation on using logos, see https://shields.io/docs/logos
-
-## Contributing Logos
-
-Our preferred way to consume icons is via the SimpleIcons logo. As a first port of call, we encourage you to contribute logos to [the SimpleIcons project][simple-icons github]. Please review their [guidance](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) before contributing.
```
Comment (chris48s) [inline doc/logos.md:7]: I think we do still need to cover this specific point in the docs somewhere

File: frontend/blog/2024-07-10-sunsetting-shields-custom-logos.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+authors:
+  name: PyvesB
+  title: Shields.io Core Team
+  url: https://github.com/PyvesB
+  image_url: https://avatars.githubusercontent.com/u/10694593
+tags: []
+---
+
+Following discussions in [#9476](https://github.com/badges/shields/issues/9476), we've gone ahead and deleted all custom logos that were maintained on the Shields.io side (bitcoin, dependabot, gitlab, npm, paypal, serverfault, stackexchange, superuser, telegram, travis), and from now on will solely rely on the [Simple-Icons project](https://github.com/simple-icons/simple-icons) to provide named logos for our badges.
+
```
Comment (chris48s) [inline frontend/blog/2024-07-10-sunsetting-shields-custom-logos.md:13]: I think it is worth explaining here that if you were using custom logos, no change is needed. We will just transparently switch you to using the corresponding simple-icon.

**Review by chris48s — COMMENTED** [review]
Mmm. I love the smell of deprecation in the morning. Thanks for picking this up.

I've flagged a couple of minor points on the documentation around this, but this is mostly good to go :+1:

File: doc/logos.md (hunk 1 above)
Comment (PyvesB) [reply doc/logos.md:7]: Added that to a small section in CONTRIBUTING.md.

**Review by chris48s — APPROVED** [review]

**Comment by PyvesB** [comment]
Will deploy and repost in an issue tomorrow.

**Pull request merged**